spec/unit/mutant/integration/rspec_spec.rb

```diff
@@ -115,10 +115,20 @@
   end
 
   before do
-    expect(RSpec::Core::ConfigurationOptions).to receive(:new).with(%w[spec --fail-fast]).and_return(options)
-    expect(RSpec::Core::Runner).to receive(:new).with(options).and_return(runner)
-    expect(RSpec).to receive(:world).with(no_args).and_return(world)
-    allow(Time).to receive(:now).and_return(Time.now)
+    expect(RSpec::Core::ConfigurationOptions).to receive(:new)
+      .with(%w[spec --fail-fast])
+      .and_return(rspec_options)
+
+    expect(RSpec::Core::Runner).to receive(:new)
+      .with(rspec_options)
+      .and_return(rspec_runner)
+
+    expect(RSpec).to receive(:world)
+      .with(no_args)
+      .and_return(world)
+
+    allow(Time).to receive(:now)
```

Comment:
In situations where you are receiving and returning a value, the receive_messages method can sometimes be cleaner:

```ruby
allow(Time).to receive_messages(now: Time.now)
```

__NOTE:__ this may apply to other specs in this PR.